.pre-commit-config.yaml

```diff
@@ -3,6 +3,6 @@ repos:
     rev: 23.10.0
     hooks:
       - id: pyink
-        args: [--pyink-indentation=2, --pyink-use-majority-quotes]
-        language_version: python3.8
+        args: [--pyink-indentation=2, --pyink-use-majority-quotes, --line-length=80]
```

Comment:
Is there any reason in particular to limiting ourselves all the way to 80 characters? I feel like 100 is usually comfortable these days or even 120. And it would be much less intrusive (this changeset is pretty big for example)

Reply:
Thanks for asking! Yes, Google Style is using 80 (internal repo as well). We plan to have a sub-repo in internal workspace to handle closed source tests for NPI. We are using Cider with default formatter (apparently, we don't want to have 2 different styles) :)